### PR TITLE
Update createmail.js

### PR DESCRIPTION
**Pull request by pAditya198**
Added logos of social media and institute, and mailto .

Fixes #{ISSUE NUMBER}

### Description

_A few sentences describing the change_

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Non Functional Requirement
- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] I have performed a self-review of my own code
- [ ] All new and existing tests passed.
- [ ] Documentation

### CODE REVIEW

**Comment by pAditya198** [comment]
Screenshot (16)

**Comment by pAditya198** [comment]
I've Fixed the logo and also added the social media links with a mailto at the bottom.

File: createmail.js [hunk 1]
```diff
@@ -65,54 +65,164 @@ const createMail = function (username, email) {
...
-                    <footer>
-                        <p>E-Mail us at codingclub@iiitvadodara.ac.in</p>
-                        <h4>Find us on</h4>
-                        <a target='_blank' class='social-link' href='https://www.facebook.com/iiitvcc/'>Facebook</a> |
-                        <a target='_blank' class='social-link' href='https://twitter.com/iiitvcc'>Twitter</a>
-                    </footer>
-                </div>
-            </body>
-        </html>    
+<!DOCTYPE html>
```
Comment (rossoskull) [inline createmail.js]: The HTML declaration and some style code are repeated, please rectify this.

**Comment by rossoskull** [comment]
Please resolve this too -

Screenshot from 2019-04-13 16-47-40

File: createmail.js (hunk 1 above)
Comment (pAditya198) [reply createmail.js]: I will see to it sir

**Pull request merged**